testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/resource/param/HeaderParamsAsPrimitivesTest.java

```diff
@@ -398,11 +398,11 @@ public void testGetByte() {
          e.printStackTrace(new PrintWriter(errors));
          fail("resourceHeaderPrimitives.doGet() failed:\n" + errors.toString());
       }
-      resourceHeaderPrimitiveWrappers.doGet(new Byte((byte) 127));
+      resourceHeaderPrimitiveWrappers.doGet(Byte.valueOf((byte) 127));

```

Comment:
This applies to the changes in this test, I think they should just all be their actual primitive values. The test name indicates "Primitive" yet we're testing the object not a primitive.